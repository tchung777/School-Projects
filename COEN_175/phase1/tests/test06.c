int printf(char *, ...);

int main(void)
{
    printf("hello world\nhow are you today? this string is reallylong");
}

void foo(int An_Identifier_To_End_All_Identifiers)
{
    auto char *p = "Let's see if \"quotes\" work in strings.";
}
