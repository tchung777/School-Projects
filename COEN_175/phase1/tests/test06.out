keyword int
identifier printf
operator (
keyword char
operator *
operator ,
operator ...
operator )
operator ;
keyword int
identifier main
operator (
keyword void
operator )
operator {
identifier printf
operator (
string "hello world\nhow are you today? this string is reallylong"
operator )
operator ;
operator }
keyword void
identifier foo
operator (
keyword int
identifier An_Identifier_To_End_All_Identifiers
operator )
operator {
keyword auto
keyword char
operator *
identifier p
operator =
string "Let's see if \"quotes\" work in strings."
operator ;
operator }
